examples/tcp-client/tcp_client.description

```diff
@@ -0,0 +1,4 @@
+// The TCP Client is used to connect to a remote TCP server.
+// This sample demonstrates how the TCP socket listener service interacts with the TCP client.<br/><br/>
```

Comment:
This sample is supposed to say how a TCP client interacts with a TCP server right? The sample is talking only about the TCP client, it shouldn't matter if it's connecting to a Ballerina based TCP server or not. Shall we reword this?

Reply:
updated to `This sample demonstrates how the TCP client interacts with the TCP server`